packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2589,6 +2590,10 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
   void didChangeDependencies() {
     super.didChangeDependencies();
 
+    _style = MediaQuery.boldTextOf(context)
+        ? widget.style.merge(const TextStyle(fontWeight: FontWeight.bold))
```

Comment:
Sorry I didn't notice this also uses widget.style, we may also need to do this in didUpdateWidget.

Reply:
oh nvm, you updated it

Reply:
Yeah, thankfully we have test coverage :)